layouts/_default/single.html

```diff
@@ -2,6 +2,9 @@
   <article class="post">
     <header class="post-header">
       <h1 class ="post-title">{{ .Title }}</h1>
+      {{- if .Description }}
+      <h4 class ="post-description">{{ .Description }}</h4>
```

Comment:
This would be nice to have in the post index as well.
Also I am not a front end guy but AFAIK you can't skip headings like that so it should be ideally an h2, more info here https://www.boia.org/blog/is-going-from-h1-to-h3-okay-for-wcag-conformance